exporter/datasetexporter/logs_exporter_test.go

```diff
@@ -224,6 +222,37 @@ func TestBuildEventFromLog(t *testing.T) {
 		ld,
 		lr.ResourceLogs().At(0).Resource(),
 		lr.ResourceLogs().At(0).ScopeLogs().At(0).Scope(),
+		newDefaultLogsSettings(),
+	)
+
+	assert.Equal(t, expected, was)
+}
+
+func TestBuildEventFromLogExportResources(t *testing.T) {
```

Comment:
In scenarios like that, I like to have test cases for 3 scenarios:

1) Implicit default value
2) Default value explicitly specified in the config
3) Default value is overridden in the config

It looks like there is already 1) and 3), but may also be a good idea to cover 2) as well.

Reply:
@tomaz-s1 what is the added value of second scenario?
1. verifies the config value if missing and also behaviour of application using false value
3. verifies the behaviour of application using true value.

Reply:
Just makes it easier to spot / catch in case default value changes (but yeah, this would also be caught by the other tests which exercises implicit default value).

Reply:
If the default changes the scenario with Implicit default value would fail so hopefully we would adjust the tests to still cover both scenarios